BikeDockingStation/schema.json

```diff
@@ -0,0 +1,67 @@
+{
```

Comment:
see https://github.com/smart-data-models/dataModel.Transportation/blob/master/BikeHireDockingStation/schema.json for potential merge